app/views/alaveteli_pro/info_request_batches/_info_request_batch.html.erb

```diff
@@ -48,7 +48,9 @@
              locals: { phases: info_request_batch.request_phases_summary } %>
 
   <% unless info_request_batch.info_requests.empty? %>
-    <div class="batch-request__requests">
+    <label class="houdini-label batch-request__show-requests" for="input1"><%= _("Show requests") %></label>
+    <input class="houdini-input" type="checkbox" id="input1">
```

Comment:
Just realised I'm going to need to make sure this id is unique because there could be several on the page at once.